packages/react-examples/jest.config.js

```diff
@@ -5,6 +5,10 @@ const config = createConfig({
   setupFiles: [path.resolve(path.join(__dirname, 'config', 'tests.js'))],
 
   snapshotSerializers: [resolveMergeStylesSerializer()],
+
+  moduleNameMapper: {
+    '@storybook/addon-docs/blocks$': '@storybook/addon-docs/dist/cjs/blocks',
```

Comment:
q: is this a know limitation (to explicitly provide CJS storybooks output) ? I mean we don't run tests on stories do we?

Reply:
Not sure about the limitation but this is a known issue (see [here](https://github.com/storybookjs/storybook/issues/13478)) that's supposed to have been fixed in `6.3.x`. Tests were written for the `utils.tsx` file in react-examples/react-components/Migrations (see file [here](https://github.com/microsoft/fluentui/blob/master/packages/react-examples/src/react-components/Migrations/utils.test.tsx)) which caused the initial failure.